intro/numpy/array_object.rst

```diff
@@ -714,8 +714,8 @@ memory and time.
 
    .. sourcecode:: pycon
 
-       >>> N_max = int(np.sqrt(len(is_prime)))
-       >>> for j in range(2, N_max):
+       >>> N_max = int(np.sqrt(len(is_prime)-1))
+       >>> for j in range(2, N_max+1):
```

Comment:
With the Python style guide, it is preferred to have spaces before and after operators.